apps/web/lib/components/Kanban.tsx

```diff
@@ -325,11 +337,24 @@ const KanbanDraggableHeader = ({
 								align="start"
 								className="md:p-1 rounded-x dark:bg-[#1B1D22] dark:border-[0.125rem] border-[#0000001A] dark:border-[#26272C] w-40"
 							>
-								{['Delete', 'Archive', 'Copy'].map((v) => (
-									<p className="hover:font-medium p-1.5 text-sm cursor-pointer" key={v}>
-										{v}
-									</p>
-								))}
+								<div
+									className="hover:font-medium p-1.5 text-sm cursor-pointer"
+									onClick={() => createTask()}
+								>
+									Create Task
+								</div>
+								<div
+									className="hover:font-medium p-1.5 text-sm cursor-pointer"
+									onClick={() => toggleColumn(title, true)}
+								>
+									Collapse Column
+								</div>
+								{/* <div
+									className="hover:font-medium p-1.5 text-sm cursor-pointer"
```

Comment:
@Anishali2 it's not "Edit Status", but should be just "Edit" from user perspective, because user will edit column settings, not "Status" really, at least how I explained it before... Anyway, I see it commented out, you want to work in another PR on it?